### DISCUSSION TITLE
Lot of 0NaN-aN-aN

### DISCUSSION THREAD

**Discussion by mdecrocemovson** [gpbl/react-day-picker#1809] [Support]
### Bug description

I am consistently getting this warning in my console:

The specified value "0NaN-aN-aN" does not conform to the required format, "yyyy-MM-dd".

When I pass a new Date() to the DayPicker component.

### To reproduce

I cannot reproduce this within a CodeSandbox, but it happens in my project.

### Screenshots

<img width="816" alt="Screenshot 2023-06-14 at 8 13 53 AM" src="https://github.com/gpbl/react-day-picker/assets/38596045/0c87b13c-cddc-4bc0-9cd0-64dad10456ed">

**Comment by MUK-Dev** [comment]
Can you share a screenshot/code of the date object causing this?

Other than that if it's not an issue try using moment js. It will make it super easy to work with dates.
